index.html

```diff
@@ -5,6 +5,7 @@
   <meta charset="UTF-8" />
   <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <script type="module" src="/src/index.ts"></script>
```

Comment:
Esto es diferente a como lo hace el scaffolding de Vite. ¿Habéis comprobado que funciona en modo dev y en modo prod?

Reply:
En teoría funciona correctamente. De todos modos, pondremos el script igual que en la configuración inicial de Vite